Documentation/operations/upgrade.rst

```diff
@@ -330,13 +289,353 @@ Annotations:
   Users may opt to disable mTLS by using the following Helm options when
   upgrading (strongly discouraged):
 
-  - ``global.hubble.tls.enabled=false``
-  - ``global.hubble.tls.auto.enabled=false``
+  - ``hubble.tls.enabled=false``
+  - ``hubble.tls.auto.enabled=false``
 * Cilium has upgraded its CRDs to v1, from v1beta1. Users must run the
   pre-flight checker mentioned above. The pre-flight check will ensure the
   custom resources installed inside the cluster are well-formed.
 * The Cilium agent is now enforcing API rate limits for certain API calls. See
   :ref:``api_rate_limiting`` for more information.
+* Cilium Helm charts have been completely re-factored. Most of the values used
+  to drive Helm charts have been re-scoped from global values to be part of a
+  single `Cilium`_, Helm chart. When upgrading from a previous version of Cilium,
+  the values will need to be provided using the new structure. In most cases the
+  prefixes of ``global.``, ``agent.``, and ``config.`` can be dropped from the
+  previously used value name. As an example, if you previously ran the command
+  ``helm install --set global.ipv4.enabled=true`` you would now run ``helm
+  install --set ipv4.enabled=true``. See the below list of new Helm values along
+  with the values previously used prior to Cilium v1.9.
```

Comment:
I think it's good to outline that global`, `agent`, `config` can be dropped like the docs describe here. 

In general it's nice to have a reference as to the old -> new values.

We should also explicitly highlight the settings that do not adhere to the simple "drop the prefix" rule. I believe there's just a handful, it will save users a lot of pain if we make those exceptions clear in a dedicated table here. I would argue this is more important than listing all of the old->new helm setting mappings.

Reply:
done!